src/probability/stopping.lean

```diff
@@ -238,17 +238,26 @@ def adapted (f : filtration ι m) (u : ι → α → β) : Prop :=
 
 namespace adapted
 
-lemma add [has_add β] [has_continuous_add β] (hu : adapted f u) (hv : adapted f v) :
+@[protected] lemma add [has_add β] [has_continuous_add β] (hu : adapted f u) (hv : adapted f v) :
```

Comment:
I know that this is not the point of the PR, but since you are modifying this line, could you change it to a `mul` version, and get the `add` lemma by using `to_additive`?